pandas/tests/reshape/test_qcut.py

```diff
@@ -130,6 +130,37 @@ def test_qcut_return_intervals():
     tm.assert_series_equal(res, exp)
 
 
+def test_qcut_labels_true():
+    # GH 13318
+    values = range(5)
+    msg = "User desired bin labels must be passed in as an argument, not just `True`"
```

Comment:
Can you add a similar test for `cut` since it also uses `_bins_to_cuts` and should now have the same behavior.

Reply:
Done